build.sbt

```
@@ -0,0 +1,28 @@
+import org.apache.commons.io.FileUtils
+
+val dottyVersion = "0.25.0-RC2"
+
+val inputDir = settingKey[File]("Directory containing the source files that will be rewitten")
+val outputDir = settingKey[File]("Directory in which the source files are rewritten")
+val checkDir = settingKey[File]("Directory that contains the expected rewritten files")
+val migration = settingKey[String]("Target Scala version - 3.0 or 3.1")
+
+val rewrites = (project in file("rewrites"))
+  .settings(
+    scalaVersion := dottyVersion,
+    migration := "3.0",
+    scalacOptions ++= Seq(s"-source:${migration.value}-migration","-rewrite"),
+    inputDir := new File(baseDirectory.value, s"src/input/scala-${migration.value}"),
```

Consider using sbt's syntax for this:
```suggestion
    inputDir := baseDirectory.value / s"src/input/scala-${migration.value}",
```
Same below.